src/tests/ftest/util/general_utils.py

```diff
@@ -1525,3 +1525,94 @@ def set_avocado_config_value(section, key, value):
+        dkey_count (int): Number of dkeys to insert.
+        akey_count (int): Number of akeys to insert.
+        base_dkey (str): Base dkey. Index numbers will be appended to it.
+        base_akey (str):Base akey. Index numbers will be appended to it.
+        base_data (str):Base data that goes inside akey. Index numbers will be appended
+            to it.
+
+    Returns:
+        tuple: Inserted objects, dkeys, akeys, and data as (ioreqs, dkeys, akeys,
+        data_list)
+
+    """
+    ioreqs = []
+    dkeys = []
+    akeys = []
+    data_list = []
+
+    container.open()
+
+    for obj_index in range(object_count):
+        # Insert object.
+        ioreqs.append(IORequest(
+            context=context, container=container.container, obj=None,
+            objtype=DaosObjClass.OC_S1))
+
+        for dkey_index in range(dkey_count):
+            # Prepare the dkey to insert into the object.
+            dkey_str = " ".join(
+                [base_dkey, str(obj_index), str(dkey_index)]).encode("utf-8")
+            dkeys.append(create_string_buffer(value=dkey_str, size=len(dkey_str)))
+
+            for akey_index in range(akey_count):
+                # Prepare the akey to insert into the dkey.
+                akey_str = " ".join(
+                    [base_akey, str(obj_index), str(dkey_index),
+                     str(akey_index)]).encode("utf-8")
+                akeys.append(create_string_buffer(value=akey_str, size=len(akey_str)))
+
+                # Prepare the data to insert into the akey.
+                data_str = " ".join(
+                    [base_data, str(obj_index), str(dkey_index),
+                     str(akey_index)]).encode("utf-8")
+                data_list.append(create_string_buffer(value=data_str, size=len(data_str)))
+                c_size = ctypes.c_size_t(ctypes.sizeof(data_list[-1]))
+
+                # Insert dkeys, akeys, and the data.
+                ioreqs[-1].single_insert(
+                    dkey=dkeys[-1], akey=akeys[-1], value=data_list[-1], size=c_size)
+
+    return (ioreqs, dkeys, akeys, data_list)
+
+
+def copy_remote_to_local(remote_file_path, test_dir, remote):
+    """Copy the given file from the server node to the local test node and retrieve
+    the original name.
+
+    Args:
+        remote_file_path (str): File path to copy to local.
+        test_dir (str): Test directory. Usually self.test_dir.
+        remote (str): Remote hostname to copy file from.
+    """
+    # Use clush --rcopy to copy the file from the remote server node to the local test
+    # node. clush will append .<server_hostname> to the file when copying.
+    args = "--rcopy {} --dest {}".format(remote_file_path, test_dir)
+    clush_command = get_clush_command(hosts=remote, args=args)
+    try:
+        run_command(command=clush_command)
+    except DaosTestError as error:
+        print("ERROR: Copying {} from {}: {}".format(remote_file_path, remote, error))
+        raise error
```

Comment:
I don't know if we should be using `print` here.
@phender Might have a suggestion.